### DISCUSSION TITLE
How to deploy version 0.6.0 using docker-compose.yml?

### DISCUSSION THREAD

**Discussion by dante1024** [docmost/docmost#541] [Help / Questions]
May I ask how to deploy version 0.6.0 using docker-compose.yml?

**Comment by coodiecool** [comment]
1. you can set your image like this:
----------------------
services:
  docmost:
    image: docmost/docmost:0.6.0
    depends_on:
      - db
      - redis
----------------------
2. or you can set your image like this:
----------------------
services:
  docmost:
    image: docmost/docmost:latest
    depends_on:
      - db
      - redis
----------------------
and using the following command to upgrade:
docker pull docmost/docmost
docker compose up --force-recreate --build docmost -d
